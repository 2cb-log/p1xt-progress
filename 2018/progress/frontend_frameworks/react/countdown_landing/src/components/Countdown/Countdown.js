import React from 'react';
import './Countdown.css';

function Countdown(props) {
  return (
    <div className="Countdown">
      Countdown
    </div>
  );

}

export default Countdown;
