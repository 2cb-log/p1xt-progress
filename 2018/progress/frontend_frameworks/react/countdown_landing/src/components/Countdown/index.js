import Countdown from './CountdownApp';

export default Countdown;