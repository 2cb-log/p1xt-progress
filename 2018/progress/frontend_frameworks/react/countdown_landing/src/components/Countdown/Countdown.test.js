import React from 'react';
import ReactDOM from 'react-dom';
import Countdown from './Countdown';
import renderer from 'react-test-renderer';

describe('<Countdown />', () => {
  const wrapper = shallow(<Countdown />);
  it('renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(<Countdown />, div);
    ReactDOM.unmountComponentAtNode(div);
  });
  it('renders correctly', () => {
    const tree = renderer
      .create(<Countdown />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
