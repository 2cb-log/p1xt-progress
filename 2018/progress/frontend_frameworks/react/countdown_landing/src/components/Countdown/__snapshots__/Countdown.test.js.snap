// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Countdown /> renders correctly 1`] = `
<div
  className="Countdown"
>
  Countdown
</div>
`;
